src/go/k8s/pkg/utils/kubernetes.go

```diff
@@ -21,3 +21,13 @@ func IsPodReady(pod *corev1.Pod) bool {
 
 	return false
 }
+
+func GetPodOrdinal(podName, clusterName string) string {
```

Comment:
nit: would be great if this returned an integer and also an error (for the first if clause)

Reply:
Changed :)